documentation/modules/ROOT/pages/index.adoc

```diff
@@ -1,92 +1,183 @@
 = Eclipse Semantic Modeling Framework
-:page-layout: tiles
+:page-layout: sections
```

Comment:
Can we name it what it is: landing-page-front?
When its possible.

Reply:
Currently not, took a note for next year, thx